### PR TITLE
Add rerun.

### PR DESCRIPTION
**Pull request by dereuromark**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Define the line ending behavior of the different file extensions
-# Set default behaviour, in case users don't have core.autocrlf set.
+# Set default behavior, in case users don't have core.autocrlf set.
```
Comment (Incognito) [inline .gitattributes]: Her royal majesty the Queen wouldn't like this one. ;)

File: src/Model/Table/QueuedJobsTable.php [hunk 2]
```diff
@@ -563,7 +563,33 @@ public function reset($id = null) {
...
+			'completed' => null,
+			'fetched' => null,
+			'progress' => 0,
+			'failed' => 0,
+			'workerkey' => null,
+			'failure_message' => null,
+		];
+		$conditions = [
+			'completed IS NOT' => null,
+			'job_type' => $type,
```
Comment (Incognito) [inline src/Model/Table/QueuedJobsTable.php]: Optionally an index here would help massive updates. I don't think we have a solid use case however

**Comment by Incognito** [comment]
Lgtm

File: .gitattributes (hunk 1 above)
Comment (dereuromark) [reply .gitattributes]: Good thing you made yourself independent from them long time ago.. ^^

File: src/Model/Table/QueuedJobsTable.php (hunk 2 above)
Comment (dereuromark) [reply src/Model/Table/QueuedJobsTable.php]: Yeah, this is not the case for many queue use cases out there.

**Pull request merged**